framework/include/fviks/FVInterfaceKernel.h

```diff
@@ -103,7 +103,7 @@ class FVInterfaceKernel : public MooseObject,
    * @return The system associated with this object. Either an undisplaced or displaced nonlinear
    * system
    */
-  const SystemBase & sys() const { return _sys; }
+  const SystemBase & sys() const { return _var1.sys(); }
```

Comment:
where do we use this? Could we rename this to `sys1()`?

Reply:
This is actually never used, at least in moose. I can run another run in civet to see if any of the apps use it.

Reply:
I suppose the only app that would use it is pronghorn

Reply:
I will remove it and run the tests again.

Reply:
Removed the whole API, we can just access the system from the variables.